### ISSUE TITLE
Javascript error "channel closed"

### ISSUE THREAD

**Issue by holmesworcester** [TryQuiet/quiet#1637]
Quiet desktop version 1.4.0 was not sending or receiving new messages

![Image](https://github.com/TryQuiet/quiet/assets/213678/24ad948a-3eb0-44fc-9834-4ffa4a40d14f)

Sending logs on Slack and Quiet.

**Comment by holmesworcester** [comment]
#1670 is possibly a duplicate of this.

**Comment by holmesworcester** [comment]
Saw this error in the logs when I shut down the app:

```
(node:216241) UnhandledPromiseRejectionWarning: Error: mainWindow is on unexpected type null
    at Timeout._onTimeout (/tmp/.mount_Quiet-sn2y6i/resources/app.asar/dist/main/main.js:427:23)
    at listOnTimeout (node:internal/timers:564:17)
    at process.processTimers (node:internal/timers:507:7)
(node:216241) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
```

**Comment by holmesworcester** [comment]
Looking into this is blocked by sentry not working #179

**Comment by holmesworcester** [comment]
@EmiM do we suspect this will be fixed by #2055 ?

**Comment by kingalg** [comment]
@holmesworcester - @Isla still sees it sometimes in logs, but it doesn't cause the app to crash. It only appears after the app has already crashed for some other reason. It's still there, but resolving it has a lower priority.
